hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -157,7 +156,7 @@ public static long getDefaultBlockSize(final FileSystem fs, final Path path) thr
     Method m = null;
     Class<? extends FileSystem> cls = fs.getClass();
     try {
-      m = cls.getMethod("getDefaultBlockSize", new Class<?>[] { Path.class });
+      m = cls.getMethod("getDefaultBlockSize", Path.class);
```

Comment:
The method `getDefaultBlockSize(Path)` has been a supported API in all the Hadoop versions we support for a very longs time. We should skip the reflection here.

Reply:
Removed the reflection part.